From 8f0a3c4ec54c22f5d54b947ba7e02730c8981b31 Mon Sep 17 00:00:00 2001
From: John Crispin <john@phrozen.org>
Date: Fri, 1 Aug 2025 14:23:23 +0200
Subject: [PATCH 10/55] include/kernel-5.4: add v5.4 hash

Adds kernel 5.4 hash to build system for legacy kernel support on older targets.

Signed-off-by: John Crispin <john@phrozen.org>
---
 include/kernel-5.4 | 7 +++++++
 1 file changed, 7 insertions(+)
 create mode 100644 include/kernel-5.4

diff --git a/include/kernel-5.4 b/include/kernel-5.4
new file mode 100644
index 0000000000..b5ae3eaa28
--- /dev/null
+++ b/include/kernel-5.4
@@ -0,0 +1,7 @@
+LINUX_VERSION-5.4 = .164
+LINUX_KERNEL_HASH-5.4.164 = a8fdb27e0e7a871086e0c78b899088c21fbc8e16fb236d220926dc4c93e0fb2d
+
+ifeq ($(CONFIG_TARGET_ipq95xx)$(CONFIG_TARGET_ipq53xx),y)
+LINUX_VERSION-5.4 = .213
+LINUX_KERNEL_HASH-5.4.213 = 53d191b59994a3b8b5c9cac7391cbf0a4186ba4bd08b9f378e2d05bfd4921f6d
+endif
-- 
2.34.1

